docs/reference/rocmcc/rocmcc.md

```diff
@@ -3,21 +3,892 @@
 ROCmCC is a Clang/LLVM-based compiler. It is optimized for high-performance computing on AMD GPUs and CPUs and supports various heterogenous programming models such as HIP, OpenMP, and OpenCL.
 
 ROCmCC is made available via two packages: rocm-llvm and rocm-llvm-alt. The differences are shown in this table:
-
+||
+|:--:|
 | <b>Table 1. rocm-llvm vs. rocm-llvm-alt</b>|
```

Comment:
Please do not use embedded HTML. Please see https://www.markdownguide.org/basic-syntax#bold

Reply:
Fixed, will need to remove similar HTML leftovers from magma_install.md